pkg/cloud/vsphere/provisioner/govmomi/delete.go

```diff
@@ -20,52 +21,69 @@ func (pv *Provisioner) Delete(ctx context.Context, cluster *clusterv1.Cluster, m
 	if cluster == nil {
 		return errors.New(constants.ClusterIsNullErr)
 	}
+	if exists, _ := pv.Exists(ctx, cluster, machine); exists {
+		err := pv.powerOffAndDestroy(ctx, cluster, machine)
+		if err != nil {
```

Comment:
This could be simplified with `if err := ...; err != nil`

Reply:
done